README.md

```diff
@@ -13,12 +13,12 @@ export YOUR_HC_API_KEY="your-secret-key-here"
 [link to documentation here]
 
 ```bash
-docker run \
-  -v opentelemetry-collector.yaml \
+docker run --rm --name collector-159pm -e YOUR_HC_API_KEY=$YOUR_HC_API_KEY \
+  -v ./opentelemetry-collector.yaml:/opentelemetry-collector.yaml \
   -p 127.0.0.1:4317:4317 \
   -p 127.0.0.1:4318:4318 \
   -p 127.0.0.1:55679:55679 \
```

Comment:
cc @bcarreno do you know which *protocol* we should open port 55679 for? I ask because we'll need to adjust this PR correctly: https://github.com/actblue/core-gitops/pull/3276/files